README.md

```diff
@@ -0,0 +1,34 @@
+# Akeneo Labs Enhanced connector bundle
+
+An export bundle primarily designed to replace the [Dnd-MagentoConnectorBundle](https://github.com/Agence-DnD/DnD-MagentoConnectorBundle) and work with [PimGento](https://github.com/Agence-DnD/PIMGento).
+
+This bundle adds some new exports to the standard ones provided by Akeneo:
+
+ - Two family export jobs, one to export complete description of the families, the other specific to PimGento that export only codes and labels.
+
+ - A new attribute export specific to PimGento, same as the standard export, but add the code of the attribute family for each attribute.
+ 
+ - A new product export, that add the possibility to choose if you want to export all products, regardless of their last modification, or only the products modified since a specific date or the last export.
+
+## Requirements
+
+ - Akeneo PIM CE or EE 1.3.x stable
+
+## Installation
+
+Install the bundle with composer:
+
+    $ php composer.phar require akeneo/enhanced-connector-bundle:1.0.*
```

Comment:
"akeneo:enhanced-connector-bundle" "~1.0"
To avoid changing the readme everytime we do a release ;)